var/spack/repos/builtin/packages/gasnet/package.py

```diff
@@ -17,7 +17,6 @@ class Gasnet(Package):
        primarily intended as a compilation target and for use by runtime library
        writers (as opposed to end users), and the primary goals are high
        performance, interface portability, and expressiveness.
-
```

Comment:
Why was this whitespace / paragraph break removed from our package description?

This is here to emphasize the notice below.

Reply:
i added the new line in https://github.com/spack/spack/pull/23382/commits/43013f54f71db4fb2d517225cad0b421564841cc.

The newline doesn't show up in `spack info` so it doesn't make a difference apart from readability. 

```
bash-3.2$ spack info gasnet
Package:   gasnet

Description:
    GASNet is a language-independent, networking middleware layer that
    provides network-independent, high-performance communication primitives
    including Remote Memory Access (RMA) and Active Messages (AM). It has
    been used to implement parallel programming models and libraries such as
    UPC, UPC++, Co-Array Fortran, Legion, Chapel, and many others. The
    interface is primarily intended as a compilation target and for use by
    runtime library writers (as opposed to end users), and the primary goals
    are high performance, interface portability, and expressiveness.
    ***NOTICE***: The GASNet library built by this Spack package is ONLY
    intended for unit-testing purposes, and is generally UNSUITABLE FOR
    PRODUCTION USE. The RECOMMENDED way to build GASNet is as an embedded
    library as configured by the higher-level client runtime package (UPC++,
    Legion, etc), including system-specific configuration.
```

Reply:
> The newline doesn't show up in spack info so it doesn't make a difference apart from readability.

This is a _defect_ in the current implementation of `spack info`, reported as issue #23831 that will hopefully be corrected soon.

Thanks for restoring the formatting that will hopefully be rendered correctly in an upcoming release.